### ISSUE TITLE
Broken link: reproducible interpredet scripts 

### ISSUE THREAD

**Issue by debrutal** [NixOS/nixos-homepage#1007]
**Introduce your stance**
Broken link on the page. 

**Describe the issue**
Link targets to https://nixos.org/guides/reproducible-scripts.html#reproducible-scripts

**Page links**
https://nixos.org/guides/ad-hoc-developer-environments.html

**Additional context**
I did not search for the actual link

![Screenshot_20230305-061146.png](https://user-images.githubusercontent.com/5798102/222942981-504d9e57-f8fe-4ffe-9ffd-26d60c511623.png)

**Comment by fricklerhandwerk** [comment]
Yeah, we just have to rewrite this website not to manually include nix.dev, or at least not automatically bump the dependency version and break all the time.

**Comment by flyfloh** [comment]
Both pages seem to not exist any more.
